pkg/controller/persistentvolume/persistentvolume_claim_binder_controller_test.go

```diff
@@ -51,6 +51,97 @@ func TestRunStop(t *testing.T) {
 	}
 }
 
+func TestRaceFix(t *testing.T) {
```

Comment:
name changed. I was able to expose the race/bug in a unit test.  I'll follow up with an improvement to the integration test and can pummel it in there against a real API server w/ watches.